external/zlib/CMakeLists.txt

```diff
@@ -33,8 +33,8 @@ else()
     message(STATUS "Could not find system zlib; building as a static library")
     include(ExternalProject)
     ExternalProject_Add(zlib
-        URL "https://zlib.net/zlib-1.2.11.tar.gz"
-        URL_HASH SHA256=c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1
+        URL "https://zlib.net/zlib-1.2.12.tar.gz"
+        URL_HASH SHA256=91844808532e5ce316b3c010929493c0244f3d37593afd6de04f71821d5136d9
```

Comment:
Thanks! This should be a separate PR so we can merge it sooner!

Reply:
Of course :)
Does it make sense to cherry-pick it for merging in master branch?